AutoCollection/CorrelationContextManager.ts

```diff
@@ -7,6 +7,8 @@ import * as DiagChannel from "./diagnostic-channel/initialization";
 
 // Don't reference modules from these directly. Use only for types.
 import * as cls from "cls-hooked";
+import TraceParent = require("../Library/TraceParent");
+import TraceState = require("../Library/TraceState");
```

Comment:
Most of the existing implementations use "Tracestate" versus "TraceState". Refer to https://github.com/census-instrumentation/opencensus-python/pull/238#discussion_r204805320.